features/crashprobe.feature

```diff
@@ -166,8 +166,15 @@ Scenario: Crash within Swift code
     And the exception "message" equals "Unexpectedly found nil while unwrapping an Optional value"
     And the exception "errorClass" equals "Fatal error"
 
-    And the "method" of stack frame 0 starts with "_T0s18_fatalErrorMessages5NeverOs12StaticStringV_A2E4fileSu4lines6UInt32V5"
-    And the "method" of stack frame 1 starts with "_T010iOSTestApp10SwiftCrashC3run"
+Scenario: Assertion failure in Swift code
```

Comment:
If it's not possible to capture messages <16 characters on Swift 4.2, I think it'd be good to add a scenario to verify this behaviour.

Reply:
Its not the _intended_ behavior, just a known limitation. It also would be specific to Xcode 10 until whenever the storage style changes again.